src/delete_old_sessions.py

```diff
@@ -0,0 +1,40 @@
+from flask_sqlalchemy import SQLAlchemy
+from app import app, db
+from app.coursegrab.models.session import Session
+import datetime
+
+""" Deletes user sessions that have been last used 10 weeks or more ago """
+def delete_old_sessions():
+  sessions = Session.query.all()
```

Comment:
maybe it would be more efficient if you can send a delete query with conditions (e.g. delete from sessions where last_used < 10 weeks old)